docs-source/content/faq/configmaps.md

```diff
@@ -0,0 +1,33 @@
+> I need to provide an instance with access to a Config Map
+
+Configuration information in Kubernetes may be provided by a 
```

Comment:
It'd be more clear IMO to change

Configuration information in Kubernetes may be provided by a

to

Configuration files can be supplied to Kubernetes pods and jobs by a